src/model.rs

```diff
@@ -108,7 +104,7 @@ impl Pod {
 // --- util types ---
 
 /// Standard metadata structure for all model instances.
-#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
```

Comment:
Perhaps leave Clone in there

Reply:
See my other [comment](https://github.com/walkerlab/orcapod/pull/42#discussion_r1827663606) on `Clone`.